lib/webpack.js

```diff
@@ -15,7 +16,9 @@ function webpack(options, callback) {
 			return webpack(options);
 		}));
 	} else if(typeof options === "object") {
-		if(!options.entry && !options.plugins) {
+		var webpackOptionsValidationErrors = validateWebpackOptions(options);
+		if(webpackOptionsValidationErrors.length) {
+			console.log("'options' validation errors:", webpackOptionsValidationErrors);
```

Comment:
We will document it in the new documentation.

https://github.com/webpack/webpack.js.org/blob/develop/content/api/node.md